src/main/java/org/folio/rest/impl/EholdingsProvidersImpl.java

```diff
@@ -74,6 +74,9 @@ public void getEholdingsProviders(String q, String sort, int page, int count, Ma
     if(!Sort.contains(sort.toUpperCase())){
       throw new ValidationException("Invalid sort parameter");
     }
+    if("".equals(q)){
+      throw new ValidationException("Search parameter cannot be empty");
+    }
     CompletableFuture.completedFuture(null)
```

Comment:
I'm not so sure we want an error in this case  - I can make a requests such as `GET eholdings/providers` against the Ruby module and I get a list of providers sorted by provider name (vendorname is rm api)

Reply:
in case of "eholdings/providers" q will be null and it will work ok, 
this is the case of "eholdings/providers?q=", for this Ruby module returns an error, and there are Postman tests that check for this error

Reply:
ahh -- got it -- thanks for clarification

Reply:
Isn't this the error message that we changed to "filter[*] cannot be empty" - If we still have "search parameter cannot be empty" - we might want to change that to filter[*] cannot be empty because "search" is a RM API term.

Reply:
The parameter name is "q", we can change error message to say "q", I just thought that if at some point this message will be shown on UI "search" will be easier to understand

Reply:
ok, makes sense!